substratevm/src/com.oracle.svm.test/src/META-INF/native-image/com.oracle.svm.test/native-image.properties

```diff
@@ -1 +1,7 @@
-Args=--initialize-at-run-time=com.oracle.svm.test --initialize-at-build-time=com.oracle.svm.test.AbstractClassSerializationTest,com.oracle.svm.test.SerializationRegistrationTest --features=com.oracle.svm.test.SerializationRegistrationTestFeature -H:+AllowVMInspection --features=com.oracle.svm.test.AbstractServiceLoaderTest$TestFeature,com.oracle.svm.test.NoProviderConstructorServiceLoaderTest$TestFeature
+Args= \
+  --initialize-at-run-time=com.oracle.svm.test \
+  --initialize-at-build-time=com.oracle.svm.test.AbstractClassSerializationTest,com.oracle.svm.test.SerializationRegistrationTest \
+  --features=com.oracle.svm.test.SerializationRegistrationTestFeature \
+  -H:+AllowVMInspection \
+  --features=com.oracle.svm.test.AbstractServiceLoaderTest$TestFeature,com.oracle.svm.test.NoProviderConstructorServiceLoaderTest$TestFeature \
+  --add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.containers=ALL-UNNAMED
```

Comment:
Yes, this is clearly the superior solution. Thanks @pejovica for the suggestion.

Reply:
@pejovica Are you sure this works for `mx native-unittest` too? The existing setup looks like it's only for `unittest`? Either way, I could get this to work. Not sure what I'm missing :(

Reply:
This is what I tried (on top of this PR): https://github.com/jerboaa/graal/commit/e5cdcd07be6e84bbf7d666cc547269ea372a7648

Reply:
Other than this suggestion (which didn't work for me), I've incorporated your feedback.

Reply:
Sorry, my bad. I assumed it would work, but we seem to be missing a few bits...

So, let's keep the current solution, and I'll add support for `@AddExports` in a separate PR.